addons/hw_drivers/main.py

```diff
@@ -95,7 +95,7 @@ def run(self):
             helpers.check_git_branch()
             helpers.generate_password()
         is_certificate_ok, certificate_details = helpers.get_certificate_status()
-        if not is_certificate_ok:
+        if not is_certificate_ok and certificate_details != 'ERR_IOT_HTTPS_CHECK_NO_SERVER':
```

Comment:
```suggestion
        if not is_certificate_ok :
   log_level = logging.INFO if certificate_details == 'ERR_IOT_HTTPS_CHECK_NO_SERVER' else logging.WARNING
   _logger.log(log_level, "An error happened when trying to get the HTTPS certificate: %s", certificate_details)
```
but make the code slightly more complicated to read (but I like the idea of keeping the logs info that the certificate was attempted)